src/data_storage/file_storage.ts

```diff
@@ -1,4 +1,4 @@
-import {FileT, FilesT, filesT} from "../types/fileT"
+import {FileT, FilesT, filesT, fileT} from "../types/fileT"
```

Comment:
Is there a need for FilesT any more on the import?

Reply:
No, will correct